content/blog/contributor-highlight-jeff-mealo/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Contributor Spotlight: Jeff Mealo"
+date: 2025-05-14T14:51:43+02:00
+draft: false
+image:
+    url: jeff.jpeg
+    attribution:
+author: fdrees
+tags:
+ - alerts
+ - dashboards
+ - chaos-engineering
+ - performance
+ - CNCF
+ - postgresql
+ - open-source
+summary: "In a mini-series on this blog we highlight the work of the community.
+  Today we meet Jeff Mealo, Senior Software Engineer at Gisual."
+---
+
+Building and maintaining and open source project takes a village. In a
+mini-series on this blog we would like to highlight the work of our
+maintainers, component owners, and members of the larger community.
+
+Today's superstar: [Jeff Mealo](https://github.com/jmealo/). Jeff is a 
+Senior Software Engineer at Gisual, where they specialize in detecting and 
+diagnosing infrastructure outages through automation. Their outage analytics 
+platform is powered by CloudNativePG, enabling organizations to quickly 
+identify network issues and achieve significant truck roll reduction. 
+Gisual both contributes engineering resources to CloudNativePG's development 
+and relies on it to keep the lights on for themselves and their customers.
+
+It's no wonder Jeff ended up in Tech, he has always had an insatiable 
+curiosity to figure out how things work:
+> "We got our first home computer at 8, and the internet at 9. The internet 
+was one of those things, and I found myself creating video game fan sites 
+using HTML and Perl from ages 9-10. One game wasn’t enough and this exploded 
+in scope to helping create the largest database of video game cheat codes 
+using PHP, and my ambitions quickly exceeded what free hosting could achieve 
+at the time. So I did what any 11 year old would do, start a computer salvage 
+operation on eBay using old gear that folks in my community had donated to me 
+to get a dedicated server and then sell enough web hosting to keep the lights 
+on!"
```

Comment:
I believe the "right" way to do multiline quotes in Markdown is with the > prefix on each line

```suggestion
> "We got our first home computer at 8, and the internet at 9. The internet
> was one of those things, and I found myself creating video game fan sites
> using HTML and Perl from ages 9-10. One game wasn’t enough and this exploded
> in scope to helping create the largest database of video game cheat codes
> using PHP, and my ambitions quickly exceeded what free hosting could achieve
> at the time. So I did what any 11 year old would do, start a computer salvage
> operation on eBay using old gear that folks in my community had donated to me
> to get a dedicated server and then sell enough web hosting to keep the lights
> on!"
```